src/components/Presentation/DetailList.vue

```diff
@@ -0,0 +1,33 @@
+<template lang="pug">
+v-list(v-if="rows.length", dense, :style="{ backgroundColor: color }")
+  v-subheader.body-2.font-weight-bold {{ title }}
```

Comment:
```suggestion
  v-subheader.subtitle-1.font-weight-bold.pl-4 {{ title }}
```

I would like to bump the size on those subheaders and vertically align them with the content, if that's alright.